<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="MessageListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>471</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="BaduRateLineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>BADU RATE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ComPortComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="TestConnectionButton">
       <property name="text">
        <string>Test Connection</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="CommandListView">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>41</y>
      <width>141</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="InvertorComboBox">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>460</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CommandLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>465</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>484</x>
      <y>460</y>
      <width>153</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="Statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
